leshan-server-redis/src/main/java/org/eclipse/leshan/server/redis/RedisSecurityStore.java

```diff
@@ -52,14 +49,10 @@ public class RedisSecurityStore implements EditableSecurityStore {
 
     private final List<SecurityStoreListener> listeners = new CopyOnWriteArrayList<>();
 
-    public RedisSecurityStore(Pool<Jedis> pool) {
-        this(pool, "SEC#EP#", "EP#PSKID");
-    }
-
-    public RedisSecurityStore(Pool<Jedis> pool, String securityInfoByEndpointPrefix, String endpointByPskIdKey) {
-        this.pool = pool;
-        this.securityInfoByEndpointPrefix = securityInfoByEndpointPrefix;
-        this.endpointByPskIdKey = endpointByPskIdKey;
+    private RedisSecurityStore(Builder builder) {
```

Comment:
If we want to allow inheritance of `RedisSecurityStore` maybe we should at least make it protected ?